src/main/java/fourth_task/FriendlyNumbers.java

```diff
@@ -0,0 +1,65 @@
+package fourth_task;
+
+import utils.InputOutputUtils;
+
+import java.io.IOException;
+import java.util.*;
+
+public class FriendlyNumbers {
+    private static final String INPUT_FILE = "src/main/resources/input_files/input_fourthtask";
+    private static final String OUTPUT_FILE = "target/logs/fourthtask_output.txt";
+
+    public static void main(String[] args) throws IOException {
+        int number = InputOutputUtils.readOnlyOneIntFromFile(INPUT_FILE);
+        HashMap<Integer, Double> map = new HashMap<>();
+        InputOutputUtils.writeToFile(getAllFriendlyPairs(map,number), OUTPUT_FILE);
+
```

Comment:
You can delete the declaration of the HashMap variable and write something like this:
`getAllFriendlyPairs(new HashMap<>(),number)`

Reply:
Right. Will do. Thanks.